knimin/lib/qiita_jira_util.py

```diff
@@ -133,6 +129,43 @@ def _create_kl_jira_project(jira_user, jira_template, study_id, title):
     return jira_project
 
 
+def _assess_replicates(prep):
+    """Renames the technical replciates in the prep information
+
+    Parameters
+    ----------
+    prep : pandas.DataFrame
+        The prep information
+
+    Returns
+    -------
+    pandas.DataFrame, list of str
+        The updates pandas dataframe and a list of replicates
+    """
+    counts = prep.sample_name.value_counts()
+    replicates = counts[counts > 1].index.tolist()
+    # Rename the samples by adding the plate id and the well.
+    # Rename the column sample_name
+    prep.rename(columns={'sample_name': 'original_sample_name'},
+                inplace=True)
+    # Create the new sample name column
+    prep['sample_name'] = prep['original_sample_name']
```

Comment:
Minor, but the prep.rename + prep['sample_name'] ... could be changed to:

prep['original_sample_name'] = prep['sample_name'].copy()